src/network/networkd-dhcp-common.c

```diff
@@ -58,10 +58,17 @@ void network_adjust_dhcp(Network *network) {
         }
 }
 
+struct DUID FALLBACK_DUID = { .type = DUID_TYPE_EN };
```

Comment:
Please make this `static`.
(I'd like to also set `const`, but it seems that requires to update logic around `link_get_duid()`. Let's do that later.)

Reply:
Yeah. I tried that too there is one place where the duid gets modified directly.